app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
```

Comment:
Could've caught `VaccineError` since those to exceptions are both its subclass as stated in the task:

```suggestion
        except VaccineError:
```